tiflash/troubleshoot-tiflash.md

```diff
@@ -94,6 +88,65 @@ Take the following steps to handle the data file corruption:
 2. Delete the related data of the TiFlash node.
 3. Redeploy the TiFlash node in the cluster.
 
+## Removing TiFlash nodes is slow
+
+Take the following steps to handle this issue:
+
+1. Check whether any table has more TiFlash replicas than the number of TiFlash nodes available after the cluster scale-in:
+
+    ```sql
+    SELECT * FROM information_schema.tiflash_replica WHERE REPLICA_COUNT >  'tobe_left_nodes';
+    ```
+
+    `tobe_left_nodes` is the number of TiFlash nodes after the scale-in.
+
+    If the query result is not empty, you need to modify the number of TiFlash replicas for the corresponding tables. This is because when the number of TiFlash replicas is greater than the number of TiFlash nodes after scale-in, PD will not move Region peers away from the TiFlash nodes to be removed, which prevents the removal of these TiFlash nodes.
```

Comment:
```suggestion
    If the query result is not empty, you need to modify the number of TiFlash replicas for the corresponding tables. This is because when the number of TiFlash replicas is greater than the number of TiFlash nodes after the scale-in, PD will not move Region peers away from the TiFlash nodes to be removed, which prevents the removal of these TiFlash nodes.
```